backend/src/main/java/com/github/alaajouri/backend/model/DTOUserData.java

```diff
@@ -0,0 +1,17 @@
+package com.github.alaajouri.backend.model;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document("user")
```

Comment:
Why the annotation? You have now 2 models who are writing to one collection. And you have another DTO model with the same attributes, why?